03-Security-In-CICD/Readme.md

```diff
@@ -240,6 +243,21 @@ what we have seen in the case of ad hoc SVT runs:
 
 [Back to top...](Readme.md#contents)
 
+
+### FAQs
+#### I have enabled AzSDK_SVTs task in my release pipeline . I am getting an error ‘The specified module 'AzSDK' was not loaded because no valid module file was found in any module directory’.  How do I resolve this issue?
+- Go to ‘AzSDK_SVTs’ task in your release definition.
+- Make sure that the check box  ‘Do not auto-update AzSDK’ is unchecked.
+ 
+#### I have enabled AzSDK_SVTs task in my release pipeline. It is taking too much time every time I queue a release, how can I reduce that time?
+- Go to ‘AzSDK_SVTs’ task in your release definition.
+- Mark the check box ‘Do not auto-update AzSDK’ as checked. 
+- This will help you save some time by not re-installing the AzSDK from scratch in every run.
+> **Note:** You will need to keep the above checkbox unchecked if you are running the AzSDK_SVTs task on any release agent for the first time OR you are running the task on Hosted VS2017 agent.
+
+
+[Back to top...](Readme.md#contents)
+
 # Security Verification Tests (SVTs) in Jenkins pipeline (Preview)
 	
 > Note : AzSDK plugin requires PowerShell to be present on Jenkins Server. Therefore, the plugin is currently supported for Windows machines only.	
```

Comment:
Do you want to add a link on how to get AzSDK started on Jenkins?

Reply:
Done with the changes.

Reply:
" Therefore, the plugin is currently supported for Windows machines only.	" This statement is not required.